### PR TITLE
fix: add package manager secrets

### PR DESCRIPTION
**Pull request by dirien**
Hi,

this PR adds the credential references to upload the different artifacts produced during the release run.

Releates to #13

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -4,29 +4,18 @@ on:
...
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+  NUGET_PUBLISH_KEY: ${{ secrets.NUGET_PUBLISH_KEY }}
+  PROVIDER: unifi
+  PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
+  PULUMI_API: https://api.pulumi.com
+  PULUMI_GO_DEP_ROOT: ${{ github.workspace }}/..
+  PULUMI_LOCAL_NUGET: ${{ github.workspace }}/nuget
+  PULUMI_TEST_OWNER: "lbrlabs"
```
Comment (ringods) [inline .github/workflows/release.yaml:16]: I don't think this line is needed.

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -4,29 +4,18 @@ on:
...
-  # YOU DO NOT NEED TO CHANGE THE PYPI_USERNAME (__token__) , IF YOU ARE USING PASSWORD AUTHENTICATION THEN YOU WILL
-  # NEED TO CHANGE TO USE THE CORRECT PASSWORD
-  PYPI_PASSWORD: ${{ YOUR PYPI PASSWORD HERE }}
-  PYPI_USERNAME: "YOUR PYPI USERNAME HERE"
-  PYPI_REPOSITORY_URL: ""
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+  NUGET_PUBLISH_KEY: ${{ secrets.NUGET_PUBLISH_KEY }}
+  PROVIDER: unifi
```
Comment (ringods) [inline .github/workflows/release.yaml:11]: Please update this line to your provider name.

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -4,29 +4,18 @@ on:
...
+  PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
```
Comment (ringods) [inline .github/workflows/release.yaml:12]: This line, as well as the line with `PULUMIPI_API` can be removed.

**Review by ringods — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yaml (hunk 1 above)
Comment (dirien) [reply .github/workflows/release.yaml:16]: removed `PULUMI_TEST_OWNER`

File: .github/workflows/release.yaml (hunk 2 above)
Comment (dirien) [reply .github/workflows/release.yaml:11]: Changed.

File: .github/workflows/release.yaml (hunk 3 above)
Comment (dirien) [reply .github/workflows/release.yaml:12]: done!

**Review by ringods — APPROVED** [review]

**Pull request merged**